ambry-api/src/main/java/com.github.ambry/network/ServerNetworkResponseMetrics.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.network;
+
+import com.codahale.metrics.Histogram;
+
+
+/**
+ * Tracks a set of metrics for a network response by a Server
+ */
+public class ServerNetworkResponseMetrics extends NetworkSendMetrics {
```

Comment:
My point is, we only care about this is a `networkSend` sent by a server, but we do not care about if this is a response or a request or something else. So, following this thought, we will have two metric classes, `ClientNetworkSendMetrics` and `ServerNetworkSendMetrics`, which track something sent by a client and something sent by a server. If you see below, the member variables do not depend on whether it is a request or a response.